dotCMS/src/test/java/com/dotcms/util/ConversionUtilsTest.java

```diff
@@ -13,6 +13,40 @@
  */
 public class ConversionUtilsTest extends UnitTestBase {
 
+    @Test
+    public void toLongFromByteCountHumanDisplaySizeTest () {
```

Comment:
this test can be rewritten using data providers and with all the different cases and just one assert